Assets/GothicVR/Scripts/Player/Menu/MenuManager.cs

```diff
@@ -0,0 +1,87 @@
+using GVR.Creator;
+using GVR.Manager;
+using GVR.Phoenix.Util;
+using UnityEngine;
+
+namespace GVR.Player.Menu
+{
+    public class MenuManager : MonoBehaviour
+    {
+        public GameObject MainMenu;
+        public GameObject SettingsMenu;
+        public GameObject TeleportMenu;
+        public GameObject MovementMenu;
+        public GameObject UIMenu;
+
+        void Awake()
+        {
+            SetSettingsValues();
+        }
+
+        public void SetSettingsValues()
+        {
+            var moveSetting = MovementMenu.FindChildRecursively("MoveSpeedSlider").GetComponent<MoveSpeedController>();
+            moveSetting.ChangeMoveSpeed(PlayerPrefs.GetFloat(ConstantsManager.I.moveSpeedPlayerPref));
+
+            var turnSetting = MovementMenu.FindChildRecursively("TurnSettingDropdown").GetComponent<TurnSettingDropdownController>();
+            turnSetting.DropdownItemSelected(PlayerPrefs.GetInt(ConstantsManager.I.turnSettingPlayerPref));
+        }
+
+        public void PlayFunction()
```

Comment:
Playfunction is a little generic. How about renaming it to LoadWaypointFromMenu or something.

Reply:
I think it was called Play in daedalus as well.